SPECS-EXTENDED/liboping/liboping.spec

```diff
@@ -85,6 +86,10 @@ LD_LIBRARY_PATH=../../src/.libs make -C bindings/perl test
 %{_mandir}/man3/ping_setopt.3*
 
 %changelog
+* Thu Jun 23 2022 Riken Maharjan <rmaharjan@microsoft.com> - 1.10.0-14
+- Patch for ncurses6.3 format issue.
+- License verified
```

Comment:
In this case we'd need to modify the `License` tag - the `COPYING` file talks about GPL 2.1. As much as possible we're trying to use [Fedora's "short names"](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses) and in this case that would be `GPLv2+` instead of `GPLv2`.

Reply:
Resolved.